{"tensorflow.python.ops.math_ops.reduce_mean": ["tensorflow.python.util.deprecation.deprecated_argument_lookup", "tensorflow.python.ops.math_ops._may_reduce_to_scalar", "tensorflow.python.ops.gen_math_ops.mean", "tensorflow.python.ops.math_ops._ReductionDims"], "tensorflow.python.ops.gen_array_ops.stop_gradient": ["tensorflow.python.eager.execute.record_gradient", "tensorflow.python.ops.gen_array_ops.stop_gradient_eager_fallback", "six.raise_from", "tensorflow.python.eager.core._status_to_exception"], "tensorflow.python.ops.nn_ops.softmax_cross_entropy_with_logits_v2": ["tensorflow.python.ops.nn_ops._ensure_xent_args", "tensorflow.python.framework.ops.name_scope.__init__", "tensorflow.python.framework.ops.convert_to_tensor", "tensorflow.python.ops.math_ops.cast", "tensorflow.python.ops.array_ops.rank", "tensorflow.python.ops.array_ops.shape", "tensorflow.python.ops.nn_ops._flatten_outer_dims", "tensorflow.python.ops.gen_nn_ops.softmax_cross_entropy_with_logits", "tensorflow.python.ops.array_ops.slice", "tensorflow.python.ops.array_ops.transpose", "tensorflow.python.ops.math_ops.subtract", "tensorflow.python.eager.context.executing_eagerly", "tensorflow.python.ops.array_ops.concat", "tensorflow.python.ops.math_ops.range"], "tensorflow.python.client.session.Session.__init__": ["tensorflow.python.client.session.BaseSession.__init__", "<builtin>.super"], "tensorflow.python.ops.math_ops.to_float": ["tensorflow.python.ops.math_ops.cast"], "tensorflow.python.ops.gen_random_ops.random_uniform": ["tensorflow.python.eager.execute.make_type", "tensorflow.python.eager.execute.make_int", "tensorflow.python.eager.execute.record_gradient", "tensorflow.python.ops.gen_random_ops.random_uniform_eager_fallback", "six.raise_from", "tensorflow.python.eager.core._status_to_exception"], "tensorflow.python.ops.array_ops.transpose": ["tensorflow.python.framework.ops.name_scope.__init__", "tensorflow.python.ops.gen_array_ops.rank", "tensorflow.python.ops.gen_math_ops._range", "tensorflow.python.eager.context.executing_eagerly"], "tensorflow.python.ops.state_ops.assign_add": ["tensorflow.python.ops.gen_state_ops.assign_add"], "tensorflow.python.ops.array_ops.expand_dims": ["tensorflow.python.util.deprecation.deprecated_argument_lookup", "tensorflow.python.ops.gen_array_ops.expand_dims"], "tensorflow.python.ops.gen_math_ops.square": ["tensorflow.python.eager.execute.record_gradient", "tensorflow.python.ops.gen_math_ops.square_eager_fallback", "six.raise_from", "tensorflow.python.eager.core._status_to_exception"], "tensorflow.python.ops.gen_nn_ops.softmax": ["tensorflow.python.eager.execute.record_gradient", "tensorflow.python.ops.gen_nn_ops.softmax_eager_fallback", "six.raise_from", "tensorflow.python.eager.core._status_to_exception"], "tensorflow.python.ops.variables.Variable.__init__": ["tensorflow.python.eager.context.executing_eagerly", "<builtin>.RuntimeError", "tensorflow.python.ops.variables.Variable._init_from_proto", "tensorflow.python.ops.variables.Variable._init_from_args", "<builtin>.ValueError"], "tensorflow.python.ops.variables.variables_initializer": ["tensorflow.python.ops.control_flow_ops.group", "tensorflow.python.eager.context.executing_eagerly"], "tensorflow.python.framework.constant_op.constant": ["tensorflow.python.eager.context.context", "tensorflow.python.framework.constant_op.convert_to_eager_tensor", "tensorflow.python.framework.tensor_shape.as_shape", "<builtin>.TypeError", "tensorflow.python.framework.constant_op._eager_reshape", "tensorflow.python.framework.constant_op._eager_identity", "tensorflow.python.framework.constant_op._eager_fill", "tensorflow.python.framework.ops.device", "<builtin>.tuple", "tensorflow.python.framework.ops.get_default_graph", "tensorflow.python.framework.tensor_util.make_tensor_proto"], "tensorflow.python.ops.array_ops.placeholder": ["tensorflow.python.eager.context.executing_eagerly", "<builtin>.RuntimeError", "tensorflow.python.ops.gen_array_ops.placeholder"], "tensorflow.python.ops.nn_ops.dropout": ["tensorflow.python.framework.ops.name_scope.__init__", "tensorflow.python.framework.ops.convert_to_tensor", "tensorflow.python.eager.context.executing_eagerly", "tensorflow.python.ops.nn_ops._get_noise_shape", "tensorflow.python.ops.random_ops.random_uniform", "<builtin>.ValueError", "<builtin>.isinstance", "tensorflow.python.ops.math_ops.div", "tensorflow.python.framework.tensor_shape.scalar", "tensorflow.python.framework.tensor_util.constant_value"], "tensorflow.python.training.gradient_descent.GradientDescentOptimizer.__init__": ["<builtin>.super"], "tensorflow.python.ops.math_ops.matmul": ["tensorflow.python.framework.ops.name_scope.__init__", "tensorflow.python.eager.context.executing_eagerly", "<builtin>.ValueError", "tensorflow.python.framework.ops.convert_to_tensor", "tensorflow.python.ops.gen_math_ops.batch_mat_mul", "tensorflow.python.ops.math_ops.conj", "tensorflow.python.ops.gen_math_ops.mat_mul", "<builtin>.isinstance", "tensorflow.python.ops.math_ops.cast", "<builtin>.len"], "tensorflow.python.util.deprecation.deprecated_argument_lookup": ["<builtin>.ValueError"], "tensorflow.python.ops.math_ops._may_reduce_to_scalar": ["tensorflow.python.framework.common_shapes.has_fully_defined_shape"], "tensorflow.python.ops.gen_math_ops.mean": ["tensorflow.python.eager.execute.make_bool", "tensorflow.python.eager.execute.record_gradient", "tensorflow.python.ops.gen_math_ops.mean_eager_fallback", "six.raise_from", "tensorflow.python.eager.core._status_to_exception"], "tensorflow.python.ops.math_ops._ReductionDims": ["<builtin>.ValueError", "tensorflow.python.framework.common_shapes.rank", "tensorflow.python.ops.math_ops.range", "tensorflow.python.framework.constant_op.constant", "<builtin>.isinstance", "tensorflow.python.ops.array_ops.rank", "numpy.arange"], "tensorflow.python.ops.gen_array_ops.stop_gradient_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.eager.execute.args_to_matching_eager", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.ops.nn_ops._ensure_xent_args": ["<builtin>.ValueError"], "tensorflow.python.framework.ops.name_scope.__init__": ["tensorflow.python.eager.context.context"], "tensorflow.python.framework.ops.convert_to_tensor": ["tensorflow.python.framework.ops.internal_convert_to_tensor"], "tensorflow.python.ops.math_ops.cast": ["tensorflow.python.framework.dtypes.as_dtype", "<builtin>.isinstance", "tensorflow.python.framework.ops.name_scope.__init__", "tensorflow.python.ops.math_ops.cast", "tensorflow.python.framework.sparse_tensor.SparseTensor.__init__", "tensorflow.python.framework.ops.convert_to_tensor", "tensorflow.python.platform.tf_logging.warn", "tensorflow.python.ops.gen_math_ops.cast"], "tensorflow.python.ops.array_ops.rank": ["tensorflow.python.ops.array_ops.rank_internal"], "tensorflow.python.ops.array_ops.shape": ["tensorflow.python.ops.array_ops.shape_internal"], "tensorflow.python.ops.nn_ops._flatten_outer_dims": ["tensorflow.python.ops.array_ops.rank", "tensorflow.python.ops.array_ops.slice", "tensorflow.python.ops.array_ops.shape", "tensorflow.python.ops.math_ops.subtract", "tensorflow.python.ops.array_ops.concat", "tensorflow.python.eager.context.executing_eagerly"], "tensorflow.python.ops.gen_nn_ops.softmax_cross_entropy_with_logits": ["tensorflow.python.eager.execute.record_gradient", "tensorflow.python.ops.gen_nn_ops.softmax_cross_entropy_with_logits_eager_fallback", "six.raise_from", "tensorflow.python.eager.core._status_to_exception"], "tensorflow.python.ops.array_ops.slice": ["tensorflow.python.ops.gen_array_ops._slice"], "tensorflow.python.ops.math_ops.subtract": ["tensorflow.python.ops.gen_math_ops.sub"], "tensorflow.python.eager.context.executing_eagerly": ["tensorflow.python.eager.context.context"], "tensorflow.python.ops.array_ops.concat": ["<builtin>.isinstance", "<builtin>.len", "tensorflow.python.framework.ops.name_scope.__init__", "tensorflow.python.ops.array_ops.identity", "tensorflow.python.framework.tensor_shape.scalar", "tensorflow.python.framework.ops.convert_to_tensor", "tensorflow.python.ops.gen_array_ops.concat_v2"], "tensorflow.python.ops.math_ops.range": ["tensorflow.python.framework.ops.name_scope.__init__", "tensorflow.python.framework.ops.convert_to_tensor", "tensorflow.python.ops.gen_math_ops._range", "<builtin>.all", "<builtin>.max", "tensorflow.python.ops.math_ops.cast"], "tensorflow.python.client.session.BaseSession.__init__": ["tensorflow.python.framework.ops.get_default_graph", "<builtin>.isinstance", "<builtin>.TypeError", "<builtin>.type", "threading.Lock", "tensorflow.python.util.compat.as_bytes"], "tensorflow.python.eager.execute.make_type": ["tensorflow.python.framework.dtypes.as_dtype", "<builtin>.TypeError", "<builtin>.repr"], "tensorflow.python.eager.execute.make_int": ["<builtin>.isinstance", "<builtin>.TypeError", "<builtin>.repr", "<builtin>.int"], "tensorflow.python.ops.gen_random_ops.random_uniform_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.eager.execute.make_type", "tensorflow.python.eager.execute.make_int", "tensorflow.python.eager.execute.args_to_matching_eager", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.ops.gen_array_ops.rank": ["tensorflow.python.eager.execute.record_gradient", "tensorflow.python.ops.gen_array_ops.rank_eager_fallback", "six.raise_from", "tensorflow.python.eager.core._status_to_exception"], "tensorflow.python.ops.gen_math_ops._range": ["tensorflow.python.eager.execute.record_gradient", "tensorflow.python.ops.gen_math_ops._range_eager_fallback", "six.raise_from", "tensorflow.python.eager.core._status_to_exception"], "tensorflow.python.ops.gen_state_ops.assign_add": ["tensorflow.python.eager.execute.make_bool", "tensorflow.python.eager.execute.record_gradient", "<builtin>.RuntimeError"], "tensorflow.python.ops.gen_array_ops.expand_dims": ["tensorflow.python.eager.execute.record_gradient", "tensorflow.python.ops.gen_array_ops.expand_dims_eager_fallback", "six.raise_from", "tensorflow.python.eager.core._status_to_exception"], "tensorflow.python.ops.gen_math_ops.square_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.eager.execute.args_to_matching_eager", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.ops.gen_nn_ops.softmax_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.eager.execute.args_to_matching_eager", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.ops.variables.Variable._init_from_proto": ["<builtin>.isinstance", "tensorflow.python.framework.ops.get_default_graph", "tensorflow.python.framework.ops.prepend_name_scope", "<builtin>.hasattr", "<builtin>.getattr", "tensorflow.python.ops.variables.Variable.SaveSliceInfo.__init__"], "tensorflow.python.ops.variables.Variable._init_from_args": ["<builtin>.ValueError", "<builtin>.callable", "<builtin>.isinstance", "<builtin>.type", "tensorflow.python.framework.ops.get_default_graph", "<builtin>.list", "tensorflow.python.framework.ops.init_scope", "tensorflow.python.eager.context.executing_eagerly", "tensorflow.python.framework.ops.add_to_collections", "<builtin>.RuntimeError", "tensorflow.python.framework.ops.name_scope.__init__", "tensorflow.python.framework.ops._name_from_scope_name", "tensorflow.python.framework.ops.convert_to_tensor", "tensorflow.python.ops.state_ops.variable_op_v2", "tensorflow.python.ops.state_ops.assign", "tensorflow.python.framework.tensor_shape.unknown_shape", "tensorflow.python.ops.variables.Variable._try_guard_against_uninitialized_dependencies", "tensorflow.python.framework.ops.device", "tensorflow.python.ops.array_ops.identity", "tensorflow.python.framework.ops.colocate_with", "tensorflow.python.util.compat.as_bytes"], "tensorflow.python.ops.control_flow_ops.group": ["tensorflow.python.eager.context.executing_eagerly", "<builtin>.ValueError", "tensorflow.python.framework.ops.name_scope.__init__", "tensorflow.python.util.nest.flatten", "<builtin>.sorted", "<builtin>.len", "tensorflow.python.ops.control_flow_ops._GroupControlDeps", "six.iterkeys", "tensorflow.python.framework.ops.control_dependencies", "<builtin>.hasattr", "<builtin>.TypeError", "<builtin>.type"], "tensorflow.python.eager.context.context": ["tensorflow.python.eager.context._initialize_context"], "tensorflow.python.framework.constant_op.convert_to_eager_tensor": ["<builtin>.isinstance", "<builtin>.TypeError", "tensorflow.python.framework.dtypes.as_dtype", "<builtin>.type"], "tensorflow.python.framework.tensor_shape.as_shape": ["<builtin>.isinstance", "tensorflow.python.framework.tensor_shape.TensorShape.__init__"], "tensorflow.python.framework.constant_op._eager_reshape": ["tensorflow.python.eager.execute.args_to_matching_eager"], "tensorflow.python.framework.constant_op._eager_fill": ["tensorflow.python.framework.constant_op.convert_to_eager_tensor"], "tensorflow.python.framework.ops.device": ["tensorflow.python.eager.context.executing_eagerly", "<builtin>.callable", "tensorflow.python.eager.context.device", "<builtin>.RuntimeError", "tensorflow.python.framework.ops.get_default_graph"], "tensorflow.python.framework.ops.get_default_graph": ["tensorflow.python.framework.ops._DefaultGraphStack.get_default"], "tensorflow.python.framework.tensor_util.make_tensor_proto": ["<builtin>.isinstance", "tensorflow.python.framework.dtypes.as_dtype", "<builtin>.callable", "numpy.asarray", "<builtin>.getattr", "<builtin>.ValueError", "numpy.empty", "tensorflow.python.framework.tensor_util._AssertCompatible", "numpy.array", "numpy.prod", "numpy.array_equal", "<builtin>.list", "tensorflow.python.framework.tensor_util._GetDenseDimensions", "<builtin>.TypeError", "<builtin>.hasattr", "<builtin>.int", "<builtin>.tuple", "tensorflow.python.framework.tensor_shape.as_shape", "tensorflow.python.framework.tensor_util._FlattenToStrings", "tensorflow.python.util.compat.as_bytes", "<builtin>.type", "tensorflow.python.framework.tensor_util.GetNumpyAppendFn"], "tensorflow.python.ops.gen_array_ops.placeholder": ["tensorflow.python.eager.execute.make_type", "tensorflow.python.eager.execute.make_shape", "tensorflow.python.eager.execute.record_gradient", "tensorflow.python.ops.gen_array_ops.placeholder_eager_fallback", "six.raise_from", "tensorflow.python.eager.core._status_to_exception"], "tensorflow.python.ops.nn_ops._get_noise_shape": ["tensorflow.python.ops.array_ops.shape", "tensorflow.python.framework.tensor_shape.as_shape", "<builtin>.enumerate", "tensorflow.python.framework.tensor_shape.TensorShape.__init__", "<builtin>.len"], "tensorflow.python.ops.random_ops.random_uniform": ["tensorflow.python.framework.dtypes.as_dtype", "<builtin>.ValueError", "tensorflow.python.framework.ops.name_scope.__init__", "tensorflow.python.ops.random_ops._ShapeTensor", "tensorflow.python.framework.ops.convert_to_tensor", "tensorflow.python.framework.random_seed.get_seed", "tensorflow.python.ops.gen_random_ops.random_uniform_int", "tensorflow.python.ops.gen_random_ops.random_uniform"], "tensorflow.python.ops.math_ops.div": ["tensorflow.python.ops.math_ops._div_python2"], "tensorflow.python.framework.tensor_util.constant_value": ["<builtin>.isinstance", "tensorflow.python.framework.tensor_util._ConstantValue"], "tensorflow.python.ops.gen_math_ops.batch_mat_mul": ["tensorflow.python.eager.execute.make_bool", "tensorflow.python.eager.execute.record_gradient", "tensorflow.python.ops.gen_math_ops.batch_mat_mul_eager_fallback", "six.raise_from", "tensorflow.python.eager.core._status_to_exception"], "tensorflow.python.ops.math_ops.conj": ["<builtin>.isinstance", "tensorflow.python.framework.ops.name_scope.__init__", "tensorflow.python.framework.ops.convert_to_tensor", "tensorflow.python.ops.gen_math_ops.conj", "<builtin>.TypeError"], "tensorflow.python.ops.gen_math_ops.mat_mul": ["tensorflow.python.eager.execute.make_bool", "tensorflow.python.eager.execute.record_gradient", "tensorflow.python.ops.gen_math_ops.mat_mul_eager_fallback", "six.raise_from", "tensorflow.python.eager.core._status_to_exception"], "tensorflow.python.framework.common_shapes.has_fully_defined_shape": ["<builtin>.isinstance"], "tensorflow.python.eager.execute.make_bool": ["<builtin>.isinstance", "<builtin>.TypeError", "<builtin>.repr"], "tensorflow.python.ops.gen_math_ops.mean_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.eager.execute.make_bool", "tensorflow.python.eager.execute.args_to_matching_eager", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.framework.common_shapes.rank": ["<builtin>.isinstance"], "tensorflow.python.eager.execute.args_to_matching_eager": ["<builtin>.isinstance"], "tensorflow.python.framework.ops.internal_convert_to_tensor": ["tensorflow.python.eager.context.context", "<builtin>.isinstance", "tensorflow.python.framework.ops.get_default_graph", "<builtin>.RuntimeError", "tensorflow.python.framework.dtypes.as_dtype", "<builtin>.type", "<builtin>.sorted", "<builtin>.TypeError", "tensorflow.python.framework.ops._error_prefix"], "tensorflow.python.framework.dtypes.as_dtype": ["<builtin>.isinstance", "<builtin>.TypeError"], "tensorflow.python.framework.sparse_tensor.SparseTensor.__init__": ["tensorflow.python.framework.ops.name_scope.__init__", "tensorflow.python.framework.ops.convert_to_tensor", "tensorflow.python.framework.ops.internal_convert_to_tensor"], "tensorflow.python.platform.tf_logging.warn": ["tensorflow.python.platform.tf_logging._get_logger"], "tensorflow.python.ops.gen_math_ops.cast": ["tensorflow.python.eager.execute.make_type", "tensorflow.python.eager.execute.record_gradient", "tensorflow.python.ops.gen_math_ops.cast_eager_fallback", "six.raise_from", "tensorflow.python.eager.core._status_to_exception"], "tensorflow.python.ops.array_ops.rank_internal": ["tensorflow.python.framework.ops.name_scope.__init__", "<builtin>.isinstance", "tensorflow.python.ops.gen_array_ops.size", "tensorflow.python.framework.ops.convert_to_tensor", "tensorflow.python.ops.gen_array_ops.rank", "tensorflow.python.framework.constant_op.constant"], "tensorflow.python.ops.array_ops.shape_internal": ["tensorflow.python.framework.ops.name_scope.__init__", "<builtin>.isinstance", "tensorflow.python.ops.gen_math_ops.cast", "tensorflow.python.ops.gen_array_ops.shape", "tensorflow.python.eager.context.executing_eagerly", "tensorflow.python.framework.ops.convert_to_tensor", "tensorflow.python.framework.constant_op.constant"], "tensorflow.python.ops.gen_nn_ops.softmax_cross_entropy_with_logits_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.eager.execute.args_to_matching_eager", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.ops.gen_array_ops._slice": ["tensorflow.python.eager.execute.record_gradient", "tensorflow.python.ops.gen_array_ops._slice_eager_fallback", "six.raise_from", "tensorflow.python.eager.core._status_to_exception"], "tensorflow.python.ops.gen_math_ops.sub": ["tensorflow.python.eager.execute.record_gradient", "tensorflow.python.ops.gen_math_ops.sub_eager_fallback", "six.raise_from", "tensorflow.python.eager.core._status_to_exception"], "tensorflow.python.ops.array_ops.identity": ["tensorflow.python.eager.context.executing_eagerly", "tensorflow.python.framework.ops.convert_to_tensor", "tensorflow.python.ops.gen_array_ops.identity", "tensorflow.python.eager.context.context"], "tensorflow.python.ops.gen_array_ops.concat_v2": ["<builtin>.len", "tensorflow.python.eager.execute.record_gradient", "<builtin>.isinstance", "<builtin>.TypeError", "tensorflow.python.ops.gen_array_ops.concat_v2_eager_fallback", "six.raise_from", "tensorflow.python.eager.core._status_to_exception"], "tensorflow.python.util.compat.as_bytes": ["<builtin>.isinstance", "<builtin>.TypeError"], "tensorflow.python.ops.gen_array_ops.rank_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.eager.execute.args_to_matching_eager", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.ops.gen_math_ops._range_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.eager.execute.args_to_matching_eager", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.ops.gen_array_ops.expand_dims_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.eager.execute.args_to_matching_eager", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.framework.ops.prepend_name_scope": ["re.sub", "tensorflow.python.platform.tf_logging.warning"], "tensorflow.python.ops.variables.Variable.SaveSliceInfo.__init__": ["<builtin>.isinstance", "tensorflow.python.framework.ops.prepend_name_scope"], "tensorflow.python.framework.ops.init_scope": ["tensorflow.python.eager.context.executing_eagerly", "tensorflow.python.framework.ops.get_default_graph", "tensorflow.python.eager.tape.stop_recording", "<builtin>.reversed", "<builtin>.RuntimeError", "tensorflow.python.framework.ops.name_scope.__init__", "tensorflow.python.framework.ops.control_dependencies", "tensorflow.python.framework.ops._DefaultGraphStack._GetGlobalDefaultGraph", "tensorflow.python.eager.context.context"], "tensorflow.python.framework.ops.add_to_collections": ["tensorflow.python.framework.ops.get_default_graph"], "tensorflow.python.ops.state_ops.variable_op_v2": ["tensorflow.python.ops.gen_state_ops.variable_v2"], "tensorflow.python.ops.state_ops.assign": ["tensorflow.python.ops.gen_state_ops.assign"], "tensorflow.python.framework.tensor_shape.unknown_shape": ["tensorflow.python.framework.tensor_shape.TensorShape.__init__", "tensorflow.python.framework.tensor_shape.Dimension.__init__"], "tensorflow.python.ops.variables.Variable._try_guard_against_uninitialized_dependencies": ["<builtin>.isinstance", "<builtin>.TypeError", "<builtin>.set", "tensorflow.python.ops.variables.Variable._safe_initial_value_from_tensor"], "tensorflow.python.framework.ops.colocate_with": ["tensorflow.python.framework.ops._colocate_with_for_gradient"], "tensorflow.python.ops.control_flow_ops._GroupControlDeps": ["tensorflow.python.framework.ops.control_dependencies", "tensorflow.python.framework.ops.device"], "tensorflow.python.framework.ops.control_dependencies": ["tensorflow.python.eager.context.executing_eagerly", "tensorflow.python.framework.ops._NullContextmanager.__default_init__", "<builtin>.callable", "tensorflow.python.framework.ops.get_default_graph"], "tensorflow.python.eager.context._initialize_context": ["tensorflow.python.eager.context.Context.__init__"], "tensorflow.python.framework.tensor_shape.TensorShape.__init__": ["<builtin>.isinstance", "<builtin>.TypeError", "tensorflow.python.framework.tensor_shape.as_dimension", "<builtin>.iter"], "tensorflow.python.eager.context.device": ["tensorflow.python.eager.context.context"], "tensorflow.python.framework.ops._DefaultGraphStack.get_default": ["tensorflow.python.framework.ops._DefaultStack.get_default", "<builtin>.super", "tensorflow.python.framework.ops._DefaultGraphStack._GetGlobalDefaultGraph"], "tensorflow.python.framework.tensor_util._AssertCompatible": ["tensorflow.python.framework.tensor_util._FirstNotNone", "<builtin>.TypeError", "<builtin>.repr", "<builtin>.type"], "tensorflow.python.framework.tensor_util._GetDenseDimensions": ["<builtin>.isinstance", "tensorflow.python.framework.tensor_util._GetDenseDimensions", "<builtin>.len"], "tensorflow.python.framework.tensor_util._FlattenToStrings": ["<builtin>.isinstance", "tensorflow.python.framework.tensor_util._FlattenToStrings"], "tensorflow.python.framework.tensor_util.GetNumpyAppendFn": ["tensorflow.python.framework.tensor_util.GetFromNumpyDTypeDict"], "tensorflow.python.eager.execute.make_shape": ["tensorflow.python.framework.tensor_shape.as_shape", "<builtin>.TypeError", "<builtin>.ValueError"], "tensorflow.python.ops.gen_array_ops.placeholder_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.eager.execute.make_type", "tensorflow.python.eager.execute.make_shape", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.ops.random_ops._ShapeTensor": ["<builtin>.isinstance", "tensorflow.python.framework.ops.convert_to_tensor"], "tensorflow.python.framework.random_seed.get_seed": ["tensorflow.python.eager.context.executing_eagerly", "tensorflow.python.eager.context.global_seed", "tensorflow.python.framework.ops.get_default_graph", "tensorflow.python.framework.random_seed._truncate_seed", "tensorflow.python.eager.context.internal_operation_seed"], "tensorflow.python.ops.gen_random_ops.random_uniform_int": ["tensorflow.python.eager.execute.make_int", "tensorflow.python.eager.execute.record_gradient", "tensorflow.python.ops.gen_random_ops.random_uniform_int_eager_fallback", "six.raise_from", "tensorflow.python.eager.core._status_to_exception"], "tensorflow.python.ops.math_ops._div_python2": ["tensorflow.python.framework.ops.name_scope.__init__", "tensorflow.python.framework.ops.convert_to_tensor", "<builtin>.TypeError", "tensorflow.python.ops.gen_math_ops.real_div", "tensorflow.python.ops.gen_math_ops.floor_div"], "tensorflow.python.framework.tensor_util._ConstantValue": ["<builtin>.isinstance", "<builtin>.TypeError", "tensorflow.python.framework.tensor_util.MakeNdarray", "numpy.array", "numpy.prod", "numpy.ndarray", "tensorflow.python.framework.tensor_util.constant_value", "numpy.arange", "tensorflow.python.framework.dtypes.as_dtype", "numpy.concatenate", "numpy.full", "numpy.equal", "numpy.not_equal"], "tensorflow.python.ops.gen_math_ops.batch_mat_mul_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.eager.execute.make_bool", "tensorflow.python.eager.execute.args_to_matching_eager", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.ops.gen_math_ops.conj": ["tensorflow.python.eager.execute.record_gradient", "tensorflow.python.ops.gen_math_ops.conj_eager_fallback", "six.raise_from", "tensorflow.python.eager.core._status_to_exception"], "tensorflow.python.ops.gen_math_ops.mat_mul_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.eager.execute.make_bool", "tensorflow.python.eager.execute.args_to_matching_eager", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.platform.tf_logging._get_logger": ["logging.getLogger", "logging.StreamHandler", "logging.Formatter"], "tensorflow.python.ops.gen_math_ops.cast_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.eager.execute.make_type", "tensorflow.python.eager.execute.args_to_matching_eager", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.ops.gen_array_ops.size": ["tensorflow.python.eager.execute.make_type", "tensorflow.python.eager.execute.record_gradient", "tensorflow.python.ops.gen_array_ops.size_eager_fallback", "six.raise_from", "tensorflow.python.eager.core._status_to_exception"], "tensorflow.python.ops.gen_array_ops.shape": ["tensorflow.python.eager.execute.make_type", "tensorflow.python.eager.execute.record_gradient", "tensorflow.python.ops.gen_array_ops.shape_eager_fallback", "six.raise_from", "tensorflow.python.eager.core._status_to_exception"], "tensorflow.python.ops.gen_array_ops._slice_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.eager.execute.args_to_matching_eager", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.ops.gen_math_ops.sub_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.eager.execute.args_to_matching_eager", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.ops.gen_array_ops.identity": ["tensorflow.python.eager.execute.record_gradient", "tensorflow.python.ops.gen_array_ops.identity_eager_fallback", "six.raise_from", "tensorflow.python.eager.core._status_to_exception"], "tensorflow.python.ops.gen_array_ops.concat_v2_eager_fallback": ["tensorflow.python.eager.context.context", "<builtin>.isinstance", "<builtin>.TypeError", "<builtin>.len", "tensorflow.python.eager.execute.args_to_matching_eager", "<builtin>.list", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.platform.tf_logging.warning": ["tensorflow.python.platform.tf_logging._get_logger"], "tensorflow.python.framework.ops._DefaultGraphStack._GetGlobalDefaultGraph": ["tensorflow.python.framework.ops.Graph.__init__"], "tensorflow.python.ops.gen_state_ops.variable_v2": ["tensorflow.python.eager.execute.make_shape", "tensorflow.python.eager.execute.make_type", "tensorflow.python.eager.execute.make_str", "tensorflow.python.eager.execute.record_gradient", "<builtin>.RuntimeError"], "tensorflow.python.ops.gen_state_ops.assign": ["tensorflow.python.eager.execute.make_bool", "tensorflow.python.eager.execute.record_gradient", "<builtin>.RuntimeError"], "tensorflow.python.framework.tensor_shape.Dimension.__init__": ["<builtin>.isinstance", "<builtin>.TypeError", "<builtin>.int", "<builtin>.ValueError"], "tensorflow.python.ops.variables.Variable._safe_initial_value_from_tensor": ["tensorflow.python.ops.variables.Variable._safe_initial_value_from_op"], "tensorflow.python.framework.ops._colocate_with_for_gradient": ["tensorflow.python.eager.context.executing_eagerly", "tensorflow.python.framework.ops.get_default_graph", "<builtin>.isinstance", "tensorflow.python.framework.ops.device", "tensorflow.python.framework.ops._NullContextmanager.__default_init__", "<builtin>.ValueError"], "tensorflow.python.eager.context.Context.__init__": ["tensorflow.python.eager.context._EagerContext.__init__", "tensorflow.python.eager.context._ContextSwitchStack.__init__", "tensorflow.python.eager.context.Context.executing_eagerly", "threading.Lock", "<builtin>.ValueError"], "tensorflow.python.framework.tensor_shape.as_dimension": ["<builtin>.isinstance", "tensorflow.python.framework.tensor_shape.Dimension.__init__"], "tensorflow.python.framework.ops._DefaultStack.get_default": ["<builtin>.len"], "tensorflow.python.framework.tensor_util._FirstNotNone": ["<builtin>.isinstance", "tensorflow.python.framework.tensor_util._Message.__init__"], "tensorflow.python.framework.tensor_util.GetFromNumpyDTypeDict": ["six.iteritems"], "tensorflow.python.eager.context.global_seed": ["tensorflow.python.eager.context.context"], "tensorflow.python.eager.context.internal_operation_seed": ["tensorflow.python.eager.context.context"], "tensorflow.python.ops.gen_random_ops.random_uniform_int_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.eager.execute.make_int", "tensorflow.python.eager.execute.args_to_matching_eager", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.ops.gen_math_ops.real_div": ["tensorflow.python.eager.execute.record_gradient", "tensorflow.python.ops.gen_math_ops.real_div_eager_fallback", "six.raise_from", "tensorflow.python.eager.core._status_to_exception"], "tensorflow.python.ops.gen_math_ops.floor_div": ["tensorflow.python.eager.execute.record_gradient", "tensorflow.python.ops.gen_math_ops.floor_div_eager_fallback", "six.raise_from", "tensorflow.python.eager.core._status_to_exception"], "tensorflow.python.framework.tensor_util.MakeNdarray": ["numpy.prod", "tensorflow.python.framework.dtypes.as_dtype", "<builtin>.len", "numpy.array", "numpy.fromiter", "numpy.frombuffer", "numpy.repeat", "<builtin>.iter", "<builtin>.TypeError", "<builtin>.complex", "<builtin>.zip"], "tensorflow.python.ops.gen_math_ops.conj_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.eager.execute.args_to_matching_eager", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.ops.gen_array_ops.size_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.eager.execute.make_type", "tensorflow.python.eager.execute.args_to_matching_eager", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.ops.gen_array_ops.shape_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.eager.execute.make_type", "tensorflow.python.eager.execute.args_to_matching_eager", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.ops.gen_array_ops.identity_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.eager.execute.args_to_matching_eager", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.framework.ops.Graph.__init__": ["threading.RLock", "tensorflow.python.util.lock_util.GroupLock.__init__", "<builtin>.dict", "threading.local", "collections.OrderedDict", "tensorflow.python.framework.traceable_stack.TraceableStack.__init__", "<builtin>.set", "tensorflow.python.framework.ops.uid", "tensorflow.python.framework.op_def_registry.get_registered_ops", "tensorflow.python.framework.ops.Graph._use_c_api_hack", "tensorflow.python.framework.c_api_util.ScopedTFGraph.__init__"], "tensorflow.python.eager.execute.make_str": ["<builtin>.isinstance", "<builtin>.TypeError", "<builtin>.repr", "tensorflow.python.util.compat.as_bytes"], "tensorflow.python.ops.variables.Variable._safe_initial_value_from_op": ["tensorflow.python.ops.variables.Variable._find_initialized_value_for_variable", "tensorflow.python.ops.variables.Variable._safe_initial_value_from_tensor"], "tensorflow.python.eager.context._EagerContext.__init__": ["<builtin>.super", "tensorflow.python.eager.context._TensorCache.__init__"], "tensorflow.python.eager.context._ContextSwitchStack.__init__": ["<builtin>.super", "tensorflow.python.eager.context._ContextSwitchStack.push"], "tensorflow.python.ops.gen_math_ops.real_div_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.eager.execute.args_to_matching_eager", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.ops.gen_math_ops.floor_div_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.eager.execute.args_to_matching_eager", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.util.lock_util.GroupLock.__init__": ["<builtin>.ValueError", "threading.Condition", "threading.Lock"], "tensorflow.python.eager.context._TensorCache.__init__": ["collections.OrderedDict"]}